docs/git-compatibility.md

```diff
@@ -173,7 +173,7 @@ echo -n '../../../.git' > .jj/repo/store/git_target
 echo '/*' > .jj/.gitignore
 # Make the Git repository non-bare and set HEAD
 git config --unset core.bare
-jj st
+jj git export
```

Comment:
Good catch, but `jj git export` doesn't work if branches are already synchronized. Perhaps, `jj new @-` works better in order to export refs and reset `HEAD@git`.

Reply:
Right, I could confirm that running `jj git export` _before_ turning the repo into a colocated one leaves the repo in an invalid state still.

`jj new @-` does seem to work in both cases :+1: